content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -94,7 +94,7 @@ When the two are specified the result is ANDed.
 If the `.spec.selector` is specified, it must match the `.spec.template.metadata.labels`.
 Config with these not matching will be rejected by the API.
 
-### Running Pods on select Nodes
+### Running Pods on selected Nodes
```

Comment:
The existing phrasing is okay and is not grammatically incorrect or a typo.
Select & Selected are both adjectives that mean "chosen from a larger number of other things". While selected is typically used to imply a subset being chosen, select implies that they're being chosen from a predefined subset specifically listed for the purpose. In this case, the Pods being created are on the nodes that match the node affinity & not randomly. Therefore select is appropriately being used here.